CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 ## [Unreleased]
 
+## [2.12.0] - 2018-08-14
+
+### Added
+- App-autoscaler included (off by default)
+- Groot-btrfs now available
+- Enabled cloud controller security events
+- nfs-broker can now be HA
+
+### Changed
+- Realigned cf role composition more inline with upstream
+- database role combines previous mysql and mysql-proxy roles
+- diego-locket role merged into diego-api
+- log-api role combines loggregator and syslog-rlp roles
+- Renamed syslog-adapter role to adapter
+- Removed processes list from all roles
+- Removed duplicate routing_api.locket.api_location property
+- Persistent StorageClass now the default
```

Comment:
For the vagrant box only

Reply:
Will cleanup upon next changelog.md edit